addons/editor/functions/fnc_handleLoad.sqf

```diff
@@ -110,6 +110,18 @@ _ctrlTreeRecent ctrlAddEventHandler ["TreeSelChanged", {
     };
 }];
 
+// Track mouse position from mouse area control to handle the mouse being over other UI elements
+// RscDisplayCurator_mousePos from base game attempts to do this but for some reason also updates when
+// the mouse is over the mission controls group
+private _fnc_updateMousePos = {
+    params ["", "_posX", "_posY"];
+    GVAR(mousePos) = [_posX, _posY];
+};
+
+private _ctrlMouseArea = _display displayCtrl IDC_RSCDISPLAYCURATOR_MOUSEAREA;
+_ctrlMouseArea ctrlAddEventHandler ["MouseMoving", _fnc_updateMousePos];
+_ctrlMouseArea ctrlAddEventHandler ["MouseHolding", _fnc_updateMousePos];
```

Comment:
Would `"MouseMoving"` not already be enough for tracking changes?

Reply:
I think the position would not update if one of the display's panels was hidden while the mouse was over it until the mouse was moved.

Reply:
Good point! Didn't think about that scenario.